### ISSUE TITLE
Small typo in help documentation

### ISSUE THREAD

**Issue by clagms** [INTO-CPS-Association/maestro#408]
To reproduce: run `java -jar .\maestro.jar --help`

Should output:
```
Usage:


mabl [-hV] [COMMAND]

Description:

Mable for co-simulating models

Options:
  -h, --help      Show this help message and exit.
  -V, --version   Print version information and exit.
Commands:
  interpret  Interpret a specification using the build in Java interpreter. Remember to place all necessary runtime extensions in the classpath
  export     Specification export
  import     Created a specification from various import types. Remember to place all " +
                     "necessary plugin extensions in the classpath.

             Hint for sg1 import where menv shoudl be enabled. Use the follding to generate the extra input file:'jq '.parameters|keys|{"environmentParameters":.}' mm.json > menv.json'
  sigver     Utilise the scenario verifier tool to generate and verify algorithms. It is also possible to execute scenarios and extended multi-models.
```

Note the following issues:
1. The command name should not be mabl.
2. In the import command help, there's a `+` that I'm pretty sure should be a string concatenation
3. The hint `Use the follding to generate ` contains typos, and it is obscure.

**Comment by CYBWEBALI** [comment]
I can fix this issue Can you assign it to me.

**Comment by lausdahl** [comment]
fixed in 30f974feee5832e4e50d5bf9df0ff71f258feb61
